chapters/planning-data-work.tex

```diff
@@ -151,162 +469,185 @@ \subsection{Code management}
 
+% ----------------------------------------------------------------------------------------------
 \subsection{Output management}
 
-Another task that needs to be discussed with your team is the best way to manage outputs.
-A great number of them will be created during the course of a project,
-from raw outputs such as tables and graphs to final products such as presentations, papers and reports.
+The final task that needs to be discussed with your team is the best way to manage output files.
+A great number of outputs will be created during the course of a project,
+and these will include both raw outputs such as tables and graphs
+and final products such as presentations, papers and reports.
 When the first outputs are being created, agree on where to store them,
-what software to use, and how to keep track of them.
+what softwares and formats to use, and how to keep track of them.
 
 % Where to store outputs
-Decisions about storage of final outputs are made easier by technical constraints.
-As discussed above, Git is a great way to control for different versions of
-plain text files, and sync software such as Dropbox are better for binary files.
-So storing raw outputs in formats like \texttt{.tex} and \texttt{.eps} in Git and
-final outputs in PDF, PowerPoint or Word, makes sense.
-Storing plain text outputs on Git makes it easier to identify changes that affect results.
+Decisions about storage of outputs are made easier by technical constraints.
+As discussed above, version control systems like Git are a great way to manage
+plaintext files, and sync softwares such as Dropbox are better for binary files.
+Outputs will similarly come in these two formats, depending on your software.
+Binary outputs like Excel files, PDFs, PowerPoints, or Word documents can be kept in a synced folder.
+Raw outputs in plaintext formats like \texttt{.tex} and \texttt{.eps}
+can be created from most analytical software and managed with Git.
+Tracking plaintext outputs with Git makes it easier to identify changes that affect results.
 If you are re-running all of your code from the master when significant changes to the code are made,
 the outputs will be overwritten, and changes in coefficients and number of observations, for example,
-will be highlighted.
+will be highlighted for you to review.
+In fact, one of the most effective ways to check code quickly
+is simply to commit all your code and outputs using Git,
+then re-run the entire thing and examine any flagged changes in the directory.
+
+No matter what choices you make,
+you will need to make updates to your outputs quite frequently.
+And anyone who has tried to recreate a graph after a few months probably knows
+that it can be hard to remember where you saved the code that created it.
+Here, naming conventions and code organization play a key role
+in not re-writing scripts again and again.
+It is common for teams to maintain one analyisis file or folder with ``exploratory analysis'',
```

Comment:
Is _exploratory analysis_ always a draft? I think using the word _draft_ makes it more clear what this is than going into explanations like "_pieces of code that are stored only to be found again in the future,
but not cleaned up to be included in any outputs yet_"

Reply:
825da0e33d7e13898d08ab76af09ab7bfef64487